bears/xml2/XCopBear.py

```diff
@@ -0,0 +1,56 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+import re
+import itertools
```

Comment:
and alphabetically sorting imports would be also good :+1:

Reply:
(also for the coalib imports above)

Reply:
Cool will make the required changes 👍